examples/crm/src/notes/NoteCreate.tsx

```diff
@@ -96,16 +96,8 @@ export const NoteCreate = ({
                                         defaultValue={getCurrentDate()}
                                     />
                                 </Stack>
-                                <FileInput source="attachment">
-                                    {formData.attachment &&
-                                    isImage(formData.attachment.rawFile) ? (
-                                        <ImageField
-                                            source="src"
-                                            title="title"
-                                        />
-                                    ) : (
-                                        <FileField source="src" title="title" />
-                                    )}
+                                <FileInput source="attachments" multiple>
+                                    <FileField source="src" title="title" />
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** ‏I haven't been able to do a different display for images or files since I switched to multiple files. If you have a solution, I'd love to hear from you. Or is the current behavior good?

Reply:
LGTM me for now, will see if this requires further improvements in the future